src/legacy/index.ts

```diff
@@ -152,6 +154,70 @@ async function shortenNodeIds(depGraph: types.DepGraphInternal): Promise<types.D
   return builder.build();
 }
 
+async function graphToDepTree(depGraphInterface: types.DepGraph, pkgType: string): Promise<DepTree> {
+  const depGraph = (depGraphInterface as types.DepGraphInternal);
+
+  // TODO: implement cycles support
+  if (depGraph.hasCycles()) {
+    throw new Error('Conversion to DepTree does not support cyclic graphs yet');
+  }
+
+  const depTree = await buildSubtree(depGraph, depGraph.rootNodeId);
+
+  depTree.packageFormatVersion = constructPackageFormatVersion(pkgType);
+
+  const targetOS = constructTargetOS(depGraph);
+  if (targetOS) {
+    depTree.targetOS = targetOS;
+  }
+
+  return depTree;
+}
+
+function constructPackageFormatVersion(pkgType: string): string {
+  if (pkgType === 'maven') {
+    pkgType = 'mvn';
+  }
+  return `${pkgType}:0.0.1`;
+}
+
+function constructTargetOS(depGraph: types.DepGraph): { name: string; version: string; } {
+  if (depGraph.pkgManager.repositories
+    && depGraph.pkgManager.repositories.length
+    && ['apk', 'apt', 'deb', 'rpm'].indexOf(depGraph.pkgManager.name) >= 0) {
```

Comment:
minor, but `.includes()` is a bit more readable.

Reply:
need to use `indexOf` as this lib targets es5